build.assets/windows/build.ps1

```diff
@@ -165,6 +165,27 @@ function Enable-Node {
     }
 }
 
+function Install-Wintun {
+    <#
+    .SYNOPSIS
+        Downloads wintun.dll into the supplied dir
+    #>
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [string] $InstallDir,
+    )
+    begin {
+        Write-Host "::group::Installing wintun.dll to $InstallDir..."
+        New-Item -Path "$InstallDir" -ItemType Directory -Force | Out-Null
+        $WintunZipfile = "$InstallDir/wintun.zip"
+        Invoke-WebRequest -Uri https://www.wintun.net/builds/wintun-0.14.1.zip -OutFile $WintunZipfile
```

Comment:
Is there a better way to manage the version of wintun.dll? I'm not sure we'll remember to update the build script, and I doubt dependabot would pick up any security vulnerabilities this way.

Reply:
Dependabot 100% will not pick this up (same with several other less visible dependencies in the Windows build process). [For legal reasons](https://gravitational.slack.com/archives/C0351AUPJ8K/p1734469268098329?thread_ts=1734465932.932219&cid=C0351AUPJ8K), we need to get the wintun build from this URL specifically, which dependabot does not have support for. If we want to be alerted to updates, we'll need to deploy Renovate to manage this.